test/views/cls.njk

```diff
@@ -34,19 +34,50 @@
   <script type="module">
     const {onCLS} = await __testImport('{{ modulePath }}');
 
+    const queue = new Set();
```

Comment:
I think it's good to add a test for an example we show in the README, but I wonder if it's overkill to test it for every single metric.

What if instead of testing it for every metric we added a single new test page that tests how all of the metrics work together and includes tests for the examples shown in the README. WDYT?

Reply:
It's up to you. I agree they're metric specific, but personally I'd be totally comfortable with a single test that reports all of the metrics, to test them all at the same time.

That type of test would also ensure that calling all of the metric functions together doesn't cause any of them to break (we don't currently have a test for that).